selfdrive/locationd/locationd.cc

```diff
@@ -521,11 +522,25 @@ int Localizer::locationd_thread() {
     }
 
     // 100Hz publish for notcars, 20Hz for cars
-    const char* trigger_msg = sm["carParams"].getCarParams().getNotCar() ? "sensorEvents" : "cameraOdometry";
-    if (sm.updated(trigger_msg)) {
+    bool kalman_msg = false;
```

Comment:
let’s keep the trigger msg logic here the same and just pick either the gyro or accelerometer to replace sensorEvents

Reply:
done